core/app_modules/inventory.js

```diff
@@ -293,9 +292,9 @@ Object.defineProperty(inventory, 'checked_out', {
     var results = (yield client.query(query)).rows;
 
     var formatted = results.map(function(a) {
-      a.due = moment(a.due).toDate();
       a.call_number = a.call;
-      a.overdue = moment(a.due).isBefore(new Date());
+      a.due = new Date(a.due.setHours((new Date().getTimezoneOffset() > 0) ? 24 : 0)); //Compensates for bug created by being in different hemisphere
```

Comment:
Should this also set the minutes, seconds, and milliseconds like the other setHours methods in this change?